scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -79,6 +79,7 @@
     - You will need to run `GvsPrepareRangesCallset` workflow first, if it has not been run already
        - This workflow transforms the data in the vet tables into a schema optimized for callset stats creation and for calculating sensitivity and precision.
        - The `only_output_vet_tables` input should be set to `true` (the default value is `false`).
+       - The `enable_extract_table_ttl` input should be set to `true` (the default value is `false`), which will add a TTL to the tables it creates.
```

Comment:
TTL makes sense to me in this context, but do we think it should be defined here?

Reply:
and I dont know where this should live, but we set the default to 2 weeks in the wdl? do we want to document that anywhere?

Reply:
I will add that it's two weeks here.